src/index.ts

```diff
@@ -1,20 +1,28 @@
 import { h } from 'hastscript'
 import type { ShikiTransformer } from '@shikijs/types'
+import type { HTMLAttributes } from 'astro/types'
 
 export interface Options {
-  toggle?: number
+  toggle?: number;
+  button?: Omit<HTMLAttributes<'button'>, 'onclick' | 'class'> & {
```

Comment:
I think it should use `HTMLButtonElement`,
And I'm not sure the `class` attribute should be omitted.